material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2014,3 +2015,143 @@ p.quick-ref-title {
   height: unset;
   padding: 1em 2em;
 }
+
+/* Cookie Consent Styling */
+.md-consent__inner {
+  z-index: 999;
+}
+
+form.md-consent__form.md-grid.md-typeset {
+  border: var(--border-width-thin) solid var(--light-transparent-15);
+  display: flex;
+  align-items: center;
+  padding: 1.5rem;
+  border-radius: 8px;
+  max-width: 90%;
+  margin: 2em auto;
+}
+
+/* Logo Section */
+.cookie-logo img {
+  display: flex;
+  margin: 2em;
+  width: 50px;
+}
+
+/* Content Section */
+.cookie-content {
+  padding: 1em;
+  flex: 1;
+  margin-right: 2em;
+}
+
+.cookie-content h4 {
+  margin: 1em 1em auto;
+  font-size: 15px;
+}
+
+.cookie-content p {
+  margin: 0 1em auto;
+  font-size: 15px;
+  color: var(--light-transparent-60);
+}
+
+/* Divider */
+.divider-settings {
+  margin: 1.5em 1em auto;
+  height: 2px;
+  background-color: var(--light-transparent-15);
+}
+
+/* Buttons Section */
+.cookie-buttons {
+  margin-bottom: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.cookie-buttons .md-button {
+  margin: 0 2em auto;
+  width: 100%;
+  padding: 10px 45px;
+  text-align: center;
+}
+
+/* Button Colors */
+button.md-button.md-button--primary {
+  color: var(--light);
+  border: solid var(--border-width) var(--teal-light);
+  transition: background-color 0.3s ease-in-out;
+}
+
+button.md-button.md-button--primary:hover {
+  color: var(--light);
+}
+
+button.md-button.md-button--secondary {
+  background-color: var(--pink-dark-transparent-15);
+  color: var(--light);
+  border: solid var(--border-width) var(--pink-dark);
+}
+
+button.md-button.md-button--secondary:hover {
+  border: solid var(--border-width) var(--pink-dark);
+  background-color: var(--pink-dark);
+  color: var(--light);
+}
+
+.md-consent__controls label.md-button {
+  color: var(--light);
+  border: solid var(--border-width) var(--teal-light);
+  transition: background-color 0.3s ease-in-out;
+}
+
+.md-consent__controls label.md-button:hover {
+  color: var(--light);
+}
+
+/* Manage Settings */
+.md-consent__settings {
+  margin: 0 1em auto;
+  animation: consent 0.5s cubic-bezier(0.1, 0.7, 0.1, 1) both;
+}
+
+ul.task-list {
+  margin: 1.5em 0 auto;
+}
+
+/* Cookie Consent Styling for Smaller Screens  */
+@media screen and (max-width: 768px) {
```

Comment:
Let's add another one here for `max-width: 1060px` that just removes the logo altogether. we don't need that on the smaller screens

```
@media screen and (max-width: 768px) {
  .cookie-logo {
    display: none;
  }
}
```